src/GraphQL/Type/InputType/AbstractRelationInputType.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\GraphQL\Type\InputType;
+
+use GraphQL\Type\Definition\InputObjectType;
+use GraphQL\Type\Definition\Type;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Service;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Traits\ServiceTrait;
+
+class AbstractRelationInputType extends InputObjectType
+{
+    use ServiceTrait;
+
+    /**
+     * AssetInputType constructor.
+     * @param Service $graphQlService
+     * @param array $config
+     * @param array $context
+     */
+    public function __construct(Service $graphQlService, $config, $context = [])
+    {
+        $this->setGraphQLService($graphQlService);
+        $this->build($config);
+        parent::__construct($config);
+    }
+
+
+    /**
+     * @param array $config
+     * @param array $context
+     */
+    public function build(&$config)
+    {
+        $config['fields'] = [
+            'id' => [
+                'type' => Type::int()

```

Comment:
should be mandatory I guess

Reply:
so then user will have to provide ` image: {id:0} `to flush the image?

Reply:
Added check in InputProcessor https://github.com/pimcore/data-hub/pull/70/files#diff-ce28ed7c2cab2b7a0025f433d8dede3fR37